torch/_torch_docs.py

```diff
@@ -3286,6 +3286,41 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Computes the regularized lower incomplete gamma function:
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input}_i)} \int_0^{\text{other}_i} t^{\text{input}_i-1} e^{-t} dt
+
+where both :math:`\text{input}_i` and :math:`\text{other}_i` are weakly positive
```

Comment:
This still needs to define the \\Gamma function used above. 

"where `\Gamma` is the ..." followed by its mathematical formula and then "and both ..." as is currently written. That is, the definition of \\Gamma should be inserted just after the mathematical expression.

Reply:
I put the definition of gamma later because I think it reads better.

Reply:
Smart. It looks really good.